.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,6 +25,7 @@ Closes #
 - [ ] when relying on **external standards**, they are referenced through a **"see"** element
 - [ ] all properties with an [simple type](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/datatypes.html) have an example value
 - [ ] metadata.json exists with status "release"
+- [ ] generated json schema validates against example json payload
```

Comment:
is this the job of the modeler or the reviewer to initially check this?

Reply:
This needs to be performed by the reviewer. However, it would be good if this is already checked by the modeler, if possible to speed up the review process.